src/main/java/org/jetbrains/research/refactorinsight/services/MiningService.java

```diff
@@ -204,7 +205,11 @@ public void onFinished() {
           }
 
           public void run(@NotNull ProgressIndicator progressIndicator) {
-            CommitMiner.mineAtCommit(commit, innerState.refactoringsMap.map, project);
+            if (timeout) {
```

Comment:
Do we always use the timeout for mining? If yes, timeout is always `true`

Reply:
Yes, removed the timeout boolean for now since we always use it as true.